src/dotnet/APIView/APIView/Model/CodeFile.cs

```diff
@@ -270,6 +273,13 @@ public void ConvertToTreeTokenModel()
                             reviewLine.parentLine = parent;
                         }
 
+                        //Handle specific cases for C++ line with 'public:' and '{' to mark related line
```

Comment:
is there another way we can guarantee this is only for c++? my worry is that another parser might have something similar and we don't know about and they run this tool and get a behavior that is not accurate